spec/imsc-hrm.html

```diff
@@ -294,11 +292,16 @@ <h2>General</h2>
       <p>The Presentation Compositor SHALL start rendering E<sub>n</sub>:</p>
 
       <ul>
-        <li>at the presentation time of E<sub>0</sub> minus Initial Painting Delay (IPD), if n = 0; or</li>
-
-        <li>at the presentation time of E<sub>n-1</sub>, if n &gt; 0.</li>
+        <li>at the presentation time of E<sub>m</sub>, where E<sub>m</sub> is the closest ISD for which m&lt;n and E<sub>m-1</sub>
+        is not empty, if E<sub>m</sub> exists and the presentation time of E<sub>n</sub> minus that of E<sub>m</sub> is less than
```

Comment:
This is really hard text to parse. In particular, I'm not sure what the `if` applies to.

One possible reading is that we only attempt to render E<sub>n</sub> if it's presentation time is >= 1 second (== IPD) later than that of E<sub>m</sub>? That doesn't seem desirable. It's frequently the case that adjacent ISDs are activated <1s apart.

Another is that if the two ISDs are >=1s apart, then we begin rendering at E<sub>0</sub> rather than at the presentation time of E<sub>n</sub> - IPD, which is what I'd expect.

In other words, _every_ ISD should be renderable in <= IPD, right?

Reply:
> In other words, every ISD should be renderable in <= IPD, right?

Yes.